docs/format.md

```diff
@@ -1,7 +1,7 @@
 Description of YAML format
 ==========================
 
-Specification version: 1.0
```

Comment:
There are no patchlevel versions to a format. You are changing something, it's either a minor increase (if compatible) or a major increase (if incompatible)